test/port-finder-socket.test.js

```diff
@@ -46,22 +47,19 @@ function createServers (callback) {
 
       server.listen(sock, next);
       base++;
-      servers.push(server);
+      servers.push([server, socket]);
     }, callback);
 }
 
-function stopServers(callback, index) {
-  if (index < servers.length) {
-    servers[index].close(function (err) {
-      if (err) {
-        callback(err, false);
-      } else {
-        stopServers(callback, index + 1);
+function stopServers(callback) {
+  _async.each(servers, function ([server, socket], next) {
+    server.close(function () {
+      if (process.platform === 'win32' && fs.existsSync(socket)) {
+        fs.unlinkSync(socket);
       }
```

Comment:
oh I see - calling server.close doesn't work on windows? was this always something we just left dangling before? 

```js
      if (process.platform === 'win32') {
        //
        // Create 'symbolic' file on the system, so it can be later
        // found via "forever list" since the `\\.pipe\\*` "files" can't
        // be enumerated because ... Windows.
        //
        fs.openSync(sock, 'w');
```


Maybe just explain this all to me like a 5 years old - thanks

Reply:
I guess I only have 1 q then - why to variables `socket` and `sock` for the same os handle?